docs/topics/mta-web-creating-custom-migration-targets.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-custom-migration-targets_{context}"]
+= Creating custom migration targets
+
+Users with `admin` permissions can create and maintain custom rulesets (migration targets). Non-admin users then can use those migration targets for analyzing applications.
+
+.Prerequisites
+
+* You must be logged in as a user with `admin` permissions.
+
+.Procedure
+
+. In *Administration* view, click *Custom migration targets*.
+. Click *Create new*.
+. Fill in the name and description of the target.
+. In the *Image* section, upload an image file for the target's icon. The file can be in either PNG or JPEG format, up to 1 Mb. If you do not upload any file, a default icon is used.
+. In the *Custom rules* section, select either *Upload manually* or *Retrieve from a repository*.
+** If you have selected *Upload manually*, upload or drag and drop the required rule files from your local drive.
+** If you have selected *Retrieve from a repository*, proceed as follows:
+... Choose *Git* or *Subversion*.
+... Fill in *Source repository*, *Branch* and *Root path*. The root path is required because MTA ignores files that are in subdirectories.
```

Comment:
"The root path is required because MTA ignores files that are in subdirectories." - apparently this is not true.
Please see internal conversation: https://redhat-internal.slack.com/archives/C048D5SFKSN/p1680623896242839

Reply:
@mguetta1 Removed